File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/DynamicCurrencyPairsVolleyNextRequest.java
package com.aneonex.bitcoinchecker.tester.volley;

import java.util.Map;

import com.android.volley.toolbox.RequestFuture;
import com.aneonex.bitcoinchecker.tester.volley.generic.GzipVolleyRequest;

public class DynamicCurrencyPairsVolleyNextRequest extends GzipVolleyRequest<String> {

	public DynamicCurrencyPairsVolleyNextRequest(String url, RequestFuture<String> future) {
		super(url, future, future);
	}

	@Override
	protected String parseNetworkResponse(Map<String, String> headers, String responseString) throws Exception {
		return responseString;
	}

}

File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/DynamicCurrencyPairsVolleyNextRequest.kt
package com.aneonex.bitcoinchecker.tester.volley

import com.android.volley.toolbox.RequestFuture
import com.aneonex.bitcoinchecker.datamodule.model.PostRequestInfo
import com.aneonex.bitcoinchecker.tester.volley.generic.GzipVolleyRequest

class DynamicCurrencyPairsVolleyNextRequest(url: String?, postRequestInfo: PostRequestInfo?, future: RequestFuture<String>) : GzipVolleyRequest<String?>(url, postRequestInfo, future, future) {
    @Throws(Exception::class)
    override fun parseNetworkResponse(headers: Map<String?, String?>?, responseString: String?): String? {
        return responseString
    }
}